Search Functionality Completed

Xenolithes opened the pull request:
The basic autofill and database search are completed. Need to add event functionality and will be mostly complete.

tylerstanczak commented on src/Components/Search-Bar.js:
```diff
@@ -0,0 +1,41 @@
+            className="Search-Bar"
+            id="Search-Bar-Id"
+            type="search" 
+            placeholder="Search Products Here"
```
The default ebay placeholder here is "Search for anything", please update this before you merge!

Xenolithes commented:
Added more functionality for the search bar including autofill with functionality. Need to work on css and the literal button functionality however, the enter key will now trigger a product change.
I also changed the place holder.
